signerserver/application.py

```diff
@@ -180,10 +180,12 @@ def forward():
             "headers": dict(response.headers),
         }
         try:
+
             proxy_response_dict["response"]["content"] = response.content.decode('utf-8')
-            proxy_response_dict["response"]['json_content'] = response.json()
+            if proxy_response_dict["response"]["content"] != "":
+                proxy_response_dict["response"]['json_content'] = response.json()
```

Comment:
Should None always be returned as JSON? What if there is an actual text/html response that is empty?

Reply:
This condition prevents exception from being sent on Empty response.